connector_jira/models/project_task/common.py

```diff
@@ -160,6 +165,50 @@ def name_get(self):
             names.append((task_id, name))
         return names
 
+    @api.model
+    def _get_connector_jira_fields(self):
```

Comment:
@simahawk @guewen one particular thing that needs checking: I didn't quite get how mapping works since it lists fields that aren't present in `project.task` model, so I've enlisted here fields that should not be allowed to be modified.

Reply:
I don't think we could really use the mapper to get the list of fields, a predefined list seems good enough.

Reply:
And the list itself is correct?

Reply:
LG at 1st glance

Reply:
would be nice to set fields as readonly but is not straightforward and not mandatory :)

Reply:
Oh, I fear doing that :)